### PR TITLE
Add .has-navbar-fixed-somewhere support to body tag (#1485)

### PR DESCRIPTION
**Pull request by Gerhut**
This is a **bugfix**.

### Proposed solution

html.has-navbar-fixed-top cannot use rem relative to html itself #1485

### Tradeoffs

> Although it is a browser bug, move .has-navbar-fixed-top to body will be a workaround.

### Testing Done

`npm run build` successfully and effectively.

### CODE REVIEW

**Comment by jimblue** [comment]
@jgthms is this PR ok for you? could we merge?

**Comment by jimblue** [comment]
@jgthms any reason why not merging this PR?

File: docs/documentation/components/navbar.html [hunk 1]
```diff
@@ -819,7 +819,7 @@ <h4 class="message-header">Javascript toggle</h4>
           {% highlight html %}<nav class="navbar is-fixed-top">{% endhighlight %}
         </li>
         <li>
-          Add the corresponding <code>has-navbar-fixed-top</code> or <code>has-navbar-fixed-bottom</code> modifier to the <code>&lt;html&gt;</code> element to provide the appropriate padding to the page
+          Add the corresponding <code>has-navbar-fixed-top</code> or <code>has-navbar-fixed-bottom</code> modifier to the <code>&lt;body&gt;</code> element to provide the appropriate padding to the page
```
Comment (jgthms) [inline docs/documentation/components/navbar.html]: This should be "to either the <code>&lt;html&gt;</code> or the <code>&lt;body&gt;</code> element".

Comment (jimblue) [reply docs/documentation/components/navbar.html]: Thanks for the review @jgthms !
@Gerhut can you make the change so @jgthms can merge? 😄

**Comment by jgthms** [comment]
Thanks for the quick fix!

**Comment by jimblue** [comment]
thanks @Gerhut !!!! 🎆

**Comment by JonathanDWood** [comment]
has-navbar-fixed-top is not present as a css rule on the body in bulma 0.6.2

Docs state:
> Add the corresponding has-navbar-fixed-top or has-navbar-fixed-bottom modifier to either `<html>` or `<body>` element to provide the appropriate padding to the page

but the only rule present is: 
`html.has-navbar-fixed-top {
  padding-top: 3.25rem;
}`

Have I misunderstood or should there also be:
`body.has-navbar-fixed-top {
  padding-top: 3.25rem;
}`

**Comment by jimblue** [comment]
The PR have been merged, you just need to wait for next release now.

**Comment by jonathanw-adsurge** [comment]
Ah okay thanks

Sorry i'm not familiar with the convention of releasing the sass for the next release to master but not updating the css.

Is that to allow people to use the features of the next release without waiting for it?

**Comment by JonathanDWood** [comment]
Ah okay thanks

Sorry i'm not familiar with the convention of releasing the sass for the next release to master but not updating the css.

Is that to allow people to use the features of the next release without waiting for it?

**Comment by VizuaaLOG** [comment]
Master is just for development. When a release is published it'll then be compiled for people using that release.

**Pull request merged**